.github/workflows/pull-request.yml

```
@@ -59,7 +59,14 @@
         if: steps.changed-files.outputs.only_changed != 'true'
         env:
           CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
-        run: npx eslint ${CHANGED_FILES} && python .github/workflows/eslint_disable_check.py
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:4:14: Double quote to prevent globbing and word splitting

(shellcheck)